bumble/host.py

```diff
@@ -1060,14 +1051,12 @@ def on_hci_connection_complete_event(
 
             # Notify the client
             self.emit(
-                'connection',
+                'connection_complete',
```

Comment:
I think we prefer using names of 'classic_connection' and 'le_connection'? At least we should be precisely classic/le_connection(_complete).

@barbibulle